src/renderers/dom/shared/eventPlugins/__tests__/ChangeEventPlugin-test.js

```diff
@@ -129,8 +129,8 @@ describe('ChangeEventPlugin', () => {
       <input type="radio" onChange={cb} />,
     );
     setUntrackedValue(input, true);
-    ReactTestUtils.SimulateNative.click(input);
-    ReactTestUtils.SimulateNative.click(input);
+    ReactTestUtils.SimulateNative.change(input);
+    ReactTestUtils.SimulateNative.change(input);
     expect(called).toBe(1);
   });
```

Comment:
Does this drop support for clicking inputs to change them? Is this possibly breaking?